pkg/devfile/parser/parse_test.go

```diff
@@ -4320,9 +4053,17 @@ func Test_parseFromRegistry(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			gotErr := false
 			got, err := parseFromRegistry(tt.importReference, &resolutionContextTree{}, tt.tool)
+			if err != nil {
+				gotErr = true
+			}
 			if (err != nil) != (tt.wantErr != nil) {
 				t.Errorf("Test_parseFromRegistry() unexpected error: %v, wantErr %v", err, tt.wantErr)
+			} else if strings.Contains(tt.name, "should be able to parse") {
```

Comment:
what is this for?  the test should always return a failure when the `err!=nil` and `tt.wantErr!=nil` does not match. which has been checked already